src/makeRobotAccountant.js

```diff
@@ -7,6 +7,29 @@
 
 function makeRobotAccountant() {
   // write code here
+  let sum = 0;
+  let counter = 0;
+
+  return (a = 0) => {
+    sum += a;
+
+    return (b = 0) => {
+      sum += b;
+      counter += 1;
+
+      if (counter > 3 && (counter % 2) === 0) {
+        sum = 0;
+
+        return 'Bzzz... Error!';
+      } else {
```

Comment:
Think about using [ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator). It will help to make your code cleaner